packages/jellyfish-api-core/__tests__/category/governance/createCfp.test.ts

```diff
@@ -0,0 +1,109 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import BigNumber from 'bignumber.js'
+import { CFPInput, ProposalStatus, ProposalType } from '../../../src/category/governance'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+
+describe('Governance', () => {
+  const container = new MasterNodeRegTestContainer(undefined, 'defi/defichain:HEAD-a61f615')
+  const client = new ContainerAdapterClient(container)
+  let payoutAddress: string
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+
+    payoutAddress = await container.call('getnewaddress')
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should not createCfp with insufficient fund', async () => {
+    const data = {
+      title: 'Testing new community fund proposal',
+      amount: new BigNumber(100),
+      cycles: 2,
+      payoutAddress
+    }
+    const promise = client.governance.createCfp(data)
+    await expect(promise).rejects.toThrow(RpcApiError)
+    await expect(promise).rejects.toThrow('RpcApiError: \'Insufficient funds\', code: -4, method: createcfp')
+  })
+
+  it('should createCfp', async () => {
+    await container.waitForWalletCoinbaseMaturity() // Wait for wallet balance to increase in order to pay 10 DFI for fund request creation
```

Comment:
```ts
await container.waitForWalletCoinbaseMaturity() // Wait for wallet balance to increase in order to pay 10 DFI for fund request creation
```

If you need another setup to test `insufficient fund` I think you should have another describe block with another condition: `describe('Governance with insufficient fund', () => {` and setup a new setup of container.

Reply:
Ok I can do that, it will be clearer